src/util/BytesLib.sol

```diff
@@ -130,4 +130,18 @@ library BytesLib {
 
         return tempBytes16;
     }
+
+    function toHex(bytes memory _bytes) public pure returns (string memory) {
```

Comment:
Any source for this?

Reply:
Just this one: https://ethereum.stackexchange.com/questions/126899/convert-bytes-to-hexadecimal-string-in-solidity

Reply:
I can add tests if you show me how.